test_programs/execution_failure/empty_composite_array_get/src/main.nr

```diff
@@ -0,0 +1,4 @@
+fn main() {
+    let empty_array: [(Field, Field); 0] = [];
+    let _ = empty_array[0];
```

Comment:
Can you add an empty array parameter to `main`? I'd like to ensure that doesn't panic in acir-gen as well